test/intl402/Temporal/PlainDate/prototype/toLocaleString/options-conflict.js

```diff
@@ -30,8 +30,15 @@ const date = new Temporal.PlainDate(2000, 5, 2);
 
 assert.sameValue(typeof date.toLocaleString("en", { dateStyle: "short" }), "string");
 
+assert.throws(TypeError, function () {
+  date.toLocaleString("en", { timeStyle: "short" });
+}, "timeStyle conflicts with PlainDate");
+
 for (const [ option, value ] of conflictingOptions) {
   assert.throws(TypeError, function() {
     date.toLocaleString("en", { [option]: value, dateStyle: "short" });
   }, `date.toLocaleString("en", { ${option}: "${value}",  dateStyle: "short" }) throws TypeError`);
+
+  // dateStyle present but undefined does not conflict
+  date.toLocaleString("en", { [option]: value, dateStyle: undefined });
```

Comment:
Should this cover `timeStyle: undefined` as well?

Reply:
Yes, no reason why it shouldn't. Thanks for catching.